OPEN_PROBLEMS/HUMAN_READABLE_NAMING.md

```diff
@@ -3,13 +3,47 @@
 ## Short Description
 > In one sentence or paragraph.
 
+You can only have two of three properties for a name: Human-meaningful, Secure and/or Decentralized. This is [Zooko's Trilemma](https://en.wikipedia.org/wiki/Zooko%27s_triangle). Can we have all 3, or even more? Can context related to some data help solve this problem?
```

Comment:
Doesn't the very page we're linking to state that we can, indeed, have all 3 and more?

Serious question as it's unclear if we're just using the triangle as a background introduction or taking it seriously, the latter meaning we are restricting the scope to exclude said existing solutions.

Reply:
Here's the issue I have with Zooko's triangle: people often equate it to the CAP theorem, overlooking the fact that one is a proven theorem and the other is a poorly formalised conjecture in a blog post.

My understanding is that there are, in fact, solutions that achieve all three properties, _depending_ on how you define those properties ("The amount of damage a malicious entity can inflict on the system should be as low as possible." is rather meaningless). Per the Wikipedia page, both Swartz' and Szabo's papers achieve security under Byzantine assumptions. The question then becomes whether that's _enough_.

Reply:
@yiannisbot's review is correct. @jsoares would you like to add a clarification on the long description that the goal is to achieve the whole 3 and at scale?

Reply:
Your answer did not shed any light for me. Do the aforementioned solutions fail to achieve all three properties and, if so, why? Is it just a scalability issue?

While I'll admit this is also a pet peeve of mine, I'm not being pedantic: if we turn this into an RFP, the solution should not be something linked on a Wikipedia page.